Setup.json

```diff
@@ -1,13 +1,13 @@
 ﻿{
```

Comment:
No need to add this file in every check-in ;-) You can exclude it from your submission or add it to the ignore list in git.